frontend/app/containers/App/actions.js

```diff
@@ -16,19 +16,21 @@
  */
 
 import {
-  LOAD_REPOS,
-  LOAD_REPOS_SUCCESS,
-  LOAD_REPOS_ERROR,
+  LOAD_NAMESPACES,
+  LOAD_NAMESPACES_ERROR,
+  LOAD_NAMESPACES_SUCCESS, LOAD_PERMS_ERROR, LOAD_PERMS_SUCCESS,
```

Comment:
Looks like these are typically on one line, can you do that here? 
```suggestion
  LOAD_NAMESPACES_SUCCESS, 
  LOAD_PERMS_ERROR, 
  LOAD_PERMS_SUCCESS,
```